docs/source/gbd2019_models/causes/postpartum_depression/index.rst

```diff
@@ -0,0 +1,191 @@
+
+Additionally, the GBD models severity distributions and associated disability weights for major depressive disorder, summarized in the table below [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). Notably, the GBD defines major depressive disorder (MDD) as an episodic mood disorder involving the experience of one or more major depressive episode(s). Notably, the percent of case severity was determined from the US National Epidemiological Survey on Alchol and Related Conditions (conducted in two waves from 2001-2002 and 2004-2005) and the Australian National Survey of Mental Health and Wellbeing of Adults. Notably, burden due to major depressive disorder in GBD begins in the 1 to 4 year age group.
+
+.. list-table:: Major depressive disorder disability weights
+  :header-rows: 1
+
+  * - Severity
+    - Percent of cases
+    - Disability weight
+  * - Asymptomatic
+    - 13 (10, 17)
+    - 0
+  * - Mild
+    - 59 (49, 69)
+    - 0.145 (0.099, 0.209)
+  * - Moderate
+    - 17 (13, 22)
+    - 0.396 (0.267, 0.531)
+  * - Severe
+    - 10 (3, 20)
+    - 0.658 (0.477, 0.807)
+
+We will conservatively assume that postpartum depression is not a cause/contributor of maternal mortality due to suicide or other causes in our simulation. While the GBD estimates deaths due to self-harm, these estimates are not specific to the postpartum population and therefore may not be generalizable.  
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+We will not model PPD as a dynamic transition model, but rather a probabilistic condition that begins at the time of birth (see the :ref:`pregnancy model document <other_models_pregnancy>`) and persists for some specified duration. The probability of experiencing PPD will be informed by a ratio per birth from the literature. PPD will be modeled as a YLD-only cause, although this is a limitation of our model as PPD may be associated with risk of suicide.
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+- We are limited in that the rate of PPD in our model is informed from a systematic literature review and meta-analysis that is not location-, year-, or age-specific and the analysis of PPD incidence primarily conducted in high-resource settings.
+
+- We are limited in that we do not consider mortality associated with PPD in our model.
+
+- We are limited in that we assume all PPD cases persist for the same average duration of a single MDD episode. This is limited in the sense that duration may vary by MDD severity (for example, a longer duration for more severe cases), which could cause our estimation of YLDs to be biased. Additionally, we assume that duration of PPD is equal to the duration of all MDD episodes.
+
+- We assume that the onset of PPD occurs immediately following birth. However, the onset of PPD may peak around two or three months postpartum [Shorey-et-al-2018]_.
```

Comment:
Additional limitation: We assume that the GBD MDD severity distribution, which is based on analysis of high-resource settings, generalizes to the severity of PPD in our simulation population of interest.

Reply:
Yes, this is good to call out! Thanks, I will add it.